aws/index.ts

```diff
@@ -1,11 +1,12 @@
 import aws from "aws-sdk";
 import nodemailer from "nodemailer";
 
-// write to aws config
+// Update AWS Config
 aws.config.update({
     accessKeyId: process.env.AWS_ACCESS_KEY_ID,
     secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
     region: process.env.AWS_REGION,
+    signatureVersion: "v4",
```

Comment:
Is this required?

Reply:
It's not required, I tested the function with an API endpoint without `signatureVersion: "v4",` and the upload worked fine. 

I took this from the AWS documentation: The Signature Version 2 or Signature Version 4 signing process is used to authenticate your Amazon S3 API requests. Signing requests enables Amazon S3 to identify who is sending the request and protects your requests from bad actors. https://docs.aws.amazon.com/AmazonS3/latest/userguide/UsingAWSSDK.html

It seems like removing it could have a security issue later on? Not sure, I won't remove it until @mahadzaryab1  takes a look

Reply:
I think this is good to keep due to the reasons that @brandonwong368 suggested.